src/main/java/com/endava/soa_5/exception_and_io/Main.java

```diff
@@ -0,0 +1,68 @@
+package com.endava.soa_5.exception_and_io;
+
+import com.endava.soa_5.array_and_string.Abecedarian;
+import com.endava.soa_5.array_and_string.MaxIndexOfArray;
+
+import java.io.IOException;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+
+        SetProperty properties = new SetProperty();
+        String arrayInString = null, word, word1, word3;
+
+        // This part of code will get value from properties file and will return boolean value if String is abecedarian or not.
+
+        try {
```

Comment:
Why not to group in a single try and multiple catches? :/ 
for example 
try { /// blablalba } 
catch (nullptr) { //throw your exception}
catch(numberformat){//throw another exception}
catch(ioexception){//do another job}

Reply:
if for example will fail on one step it will catch the exception, but another part of the code will not execute.